Jint/Runtime/Debugger/DebugEvaluationException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Jint.Runtime.Debugger
```

Comment:
could use file-scoped namespaces for new files and set #nullable enable to reduce later work